### PR TITLE
Bumping examples to 1.12

### PR DESCRIPTION
**Pull request by anthonyshew**
### Description

Bumping dependencies in examples.

Closes TURBO-2182

### CODE REVIEW

File: examples/with-rollup/package.json [hunk 1]
```diff
@@ -1,9 +1,5 @@
 {
   "private": true,
-  "workspaces": [
-    "apps/*",
-    "packages/*"
-  ],
```
Comment (chris-olszewski) [inline examples/with-rollup/package.json:6]: I think we still want workspaces in this example?

File: examples/with-rollup/package.json [hunk 2]
```diff
@@ -1,9 +1,5 @@
...
   "scripts": {
```
Comment (chris-olszewski) [inline examples/with-rollup/package.json:3]: Also bump the turbo version specifier to `^1.12.1`

**Review by chris-olszewski — APPROVED** [review]
Stamping there's just a minor fixup

File: examples/with-rollup/package.json (hunk 1 above)
Comment (anthonyshew) [reply examples/with-rollup/package.json:6]: It's pnpm! Fixing what shouldn't have been there in the first place.

**Pull request merged**